components/TripCard.js

```diff
@@ -0,0 +1,40 @@
+import styled from "styled-components";
+
+export default function TripCard({ country }) {
+  return (
+    <CardBox image={`https://source.unsplash.com/random/?${country}`}>
+      <TextArea>
+        <CountryName>{country}</CountryName>
+      </TextArea>
+    </CardBox>
+  );
+}
+
+const CardBox = styled.button`
+  width: 20em;
+  height: 20em;
+  filter: drop-shadow(0 0 1em #a1a1a1);
```

Comment:
u could use the color var in your GlobalStyle

Reply:
Added a color var for drop-shadow. Will be useful in future tasks.